internal/kibana/route.go

```diff
@@ -80,7 +81,12 @@ func (clusterRequest *KibanaRequest) createOrUpdateKibanaConsoleLink() error {
 		return kverrors.Wrap(err, "failed to get route URL for kibana")
 	}
 
-	cl := console.NewConsoleLink(KibanaConsoleLinkName, kibanaURL, "Logging", "Observability")
+	loggingIcon, err := url.ParseRequestURI("https://github.com/openshift/elasticsearch-operator/blob/master/files/eo_icon.svg?raw=true")
+	if err != nil {
+		log.Error(err, "Invalid icon URL")
+	}
+
+	cl := console.NewConsoleLink(KibanaConsoleLinkName, kibanaURL, "Logging", loggingIcon.String(), "Observability")
```

Comment:
Sure, I kind of expected this would be an issue.

Reply:
@periklis I updated the PR with another workaround which is using a data URI representation of the icon instead of a link.
However, if `go embed` is still preferred, we will have to put the icon file in the same path as the function calling it, which is `internal/kibana/` instead of `files/`

Reply:
LGTM and in sufficient for now. We don't need go embed here. It doesn't buy us much.